ocaml/xapi/xapi_vif_helpers.ml

```diff
@@ -95,7 +95,10 @@ let valid_operations ~__context record _ref' : table =
         | `no -> set_errors Api_errors.operation_not_allowed ["VM states it does not support VIF hotplug."] [`plug; `unplug]
           (* according to xen docs PV drivers are enough for this to be possible *)
         | `unspecified when gmr.Db_actions.vM_guest_metrics_PV_drivers_detected -> ()
-        | `unspecified -> fallback ())
+        | `unspecified -> fallback ()
+        | `unknown ->
+          (* Safe fallback *)
+          set_errors Api_errors.operation_not_allowed ["VM states it does not support VIF hotplug."] [`plug; `unplug])
```

Comment:
`"VM states it does not support VIF hotplug."` --really? I think this would happen if the VM has a statement that was understood by the XenServer where the VM was running at the time, but which we don't understand in the current XenServer.
In other words, `"VM made a statement about whether it supports VIF hotplug. This statement is not known but it was neither yes nor no. It is being treated as a no."`
I suppose this might be equivalent to `"VM states it does not have unconditional support for VIF hotplug; this is treated as having no support."`

Reply:
Yep, will fix thanks.